src/docs/content.go

```diff
@@ -38,109 +35,109 @@ Enterprise Performance Computing (EPC)`
 	BuildLong  string = `
 
   IMAGE PATH:
-  
+
```

Comment:
Are we sure we're not introducing too much whitespace here in all of these help messages? We want to make sure to be outputting in a very readable form. I'm indifferent on this, but I think @GodloveD should have good insight into if this is too much whitespace

Reply:
I'm not adding a white line here, It seems, that my IDE, just removed a [tab/space] on that line, and replaced it with a clean line.

Reply:
Ahh okay, so it's not actually a change. No worries then it's already been approved by DG ;) carry on